### PR TITLE
Implement 1548

### PR DESCRIPTION
**Pull request by JeanMarie-TTS**
## PR checklist: submitters

- [x]   Link to an issue if possible. If there’s no issue, describe what your branch does. Even if there is an issue, a brief description in the PR is still useful.

- #1548 

- [ ]   List any special steps reviewers have to follow to test the PR. For example, adding a local environment variable, creating a local test file, etc.
- [ ]   For extra credit, submit a screen recording like [this one](https://github.com/GSA-TTS/FAC/pull/1821).
- [x]   Make sure you’ve merged `main` into your branch shortly before creating the PR. (You should also be merging `main` into your branch regularly during development.)
- [ ]   Make sure that whatever feature you’re adding has tests that cover the feature. This includes test coverage to make sure that the previous workflow still works, if applicable.

- No new features.

- [x]   Do manual testing locally. Our tests are not good enough yet to allow us to skip this step. If that’s not applicable for some reason, check this box.
- [ ]   Verify that no Git surgery was necessary, or, if it was necessary at any point, repeat the testing after it’s finished.
- [ ]   Once a PR is merged, keep an eye on it until it’s deployed to dev, and do enough testing on dev to verify that it deployed successfully, the feature works as expected, and the happy path for the broad feature area (such as submission) still works.

## PR checklist: reviewers

- [ ]   Pull the branch to your local environment and run `make docker clean; make docker-first-run && docker compose up`; then run `docker compose exec web /bin/bash -c "python manage.py test"`
- [ ]   Manually test out the changes locally, or check this box to verify that it wasn’t applicable in this case.
- [ ]   Check that the PR has appropriate tests. Look out for changes in HTML/JS/JSON Schema logic that may need to be captured in Python tests even though the logic isn’t in Python.
- [ ]   Verify that no Git surgery is necessary at any point (such as during a merge party), or, if it was, repeat the testing after it’s finished.
        
The larger the PR, the stricter we should be about these points.

### CODE REVIEW

File: backend/audit/etl.py [hunk 1]
```diff
@@ -333,76 +340,98 @@ def load_general(self):
...
-                            "secondary_auditor_address_state"
-                        ],
-                        address_zipcode=secondary_auditor[
-                            "secondary_auditor_address_zipcode"
-                        ],
-                    )
-                    sec_auditor.save()
+        if not secondary_auditors:
+            general.secondary_auditors_exist = False
+            general.save
```
Comment (tadhg-ohiggins) [inline backend/audit/etl.py:349]:
```suggestion
            general.save()
```
I assume this should be a method invocation.

Comment (JeanMarie-TTS) [reply backend/audit/etl.py]: This line of code has been removed.

File: backend/audit/test_etl.py [hunk 2]
```diff
@@ -357,10 +378,12 @@ def test_load_all(self):
         self.etl.load_all()
         self.assertLess(len_general, len(General.objects.all()))
         self.assertLess(len_captext, len(CapText.objects.all()))
+        self.assertLess(len_additional_ueis, len(AdditionalUei.objects.all()))
+        self.assertLess(len_additional_eins, len(AdditionalEin.objects.all()))
 
     def test_load_all_with_errors_1(self):
-        """If we encounter a KeyError on General (the first table to be loaded), we
-        should still load all the other tables, but nothing should be loaded to General.
+        """IWe should not encounter a key error in general,
```
Comment (sambodeme) [inline backend/audit/test_etl.py:385]:
```suggestion
        """We should not encounter a key error in general,
```

File: backend/audit/test_etl.py [hunk 3]
```diff
@@ -374,15 +397,14 @@ def test_load_all_with_errors_1(self):
             corrective_action_plan=self._fake_corrective_action_plan(),
             secondary_auditors=self._fake_secondary_auditors(),
             additional_ueis=self._fake_additional_ueis(),
-            # audit_information=self._fake_audit_information(),  # TODO: Uncomment when SingleAuditChecklist adds audit_information
```
Comment (sambodeme) [inline backend/audit/test_etl.py:377]: Any reason why this is removed here but used [here](https://github.com/GSA-TTS/FAC/pull/1912/files#diff-b594cf1258aab056b2cf43b2bfeec18a7c58b02dd6c0455c4f2683b1aad4cc6dR372) ?

File: backend/audit/test_etl.py [hunk 4]
```diff
@@ -13,6 +14,8 @@
     FindingText,
     CapText,
     SecondaryAuditor,
+    AdditionalUei,
```
Comment (sambodeme) [inline backend/audit/test_etl.py:17]: How about using `AdditionalUeis` and `AdditionalEins` to be consistent with the existing naming in the code

**Review by sambodeme — APPROVED** [review]

**Pull request merged**